components/nav.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import Link from 'next/link'
+
+export default class Form extends React.Component {
+  state = {
+    title: "",
+    description: "",
+    textField: ""
+  };
+
+  change = e => {
+    this.props.onChange({ [e.target.name]: e.target.value });
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  onSubmit = e => {
+    e.preventDefault();
+    // this.props.onSubmit(this.state);
+    this.setState({
+      title: "",
+      description: "",
+      textField: ""
+    });
+    this.props.onChange({
+      title: "",
+      Description: "",
+      textField: ""
+    });
+  };
+
+  render() {
+    return (
+      <form>
+        <input
+          name="title"
+          placeholder="Title"
+          value={this.state.title}
+          onChange={e => this.change(e)}
+        />
+        <br />
+        <input
+          name="description"
+          placeholder="Description"
+          value={this.state.description}
+          onChange={e => this.change(e)}
+        />
+        <br />
+        <input
+          name="textField"
+          placeholder="Text Field"
+          value={this.state.textField}
+          onChange={e => this.change(e)}
+        />
+        <br />
```

Comment:
A block element would be more appropriate than using BR (line break )  for layout/styling here.  Labels would also be very useful and good to use for the inputs.
I'd suggest taking a look here for a good reference on form HTML structuring (does not cover everything of course, but it's a good starting point!) 
https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/How_to_structure_an_HTML_form

Reply:
Updated